<launch>
    <node pkg="rr_avc" type="color_detector_avc" name="color_detector_avc" output="screen">
        <param name="image_topic" type="string" value="/camera/image_raw"/>
        <param name="pub_topic" type="string" value="/colors_img"/>
        <param name="image_width" type="int" value="160"/>
        <param name="image_height" type="int" value="90"/>
        <param name="mask_px_top" type="int" value="45"/>
        <param name="mask_px_bottom" type="int" value="15"/>
        <param name="blur_strength" type="int" value="5"/>
    </node>
</launch>
